web/db/actions/Post.ts

```diff
@@ -82,6 +90,54 @@ async function getResizedUploadUrl(uuid: string): Promise<string> {
   return `${consts.baseUrl}/api/resizedUpload/${token}`;
 }
 
+async function deleteAttachments(keysToDelete: string[]) {
+  let numTries = 1;
+  const MAXTRIES = 3;
+  const objectsToDelete = keysToDelete.map((keyToDelete) => ({
+    Key: keyToDelete,
+  }));
+  const deleteObjectsCommand = new DeleteObjectsCommand({
+    Bucket: consts.storageBucket,
+    Delete: { Objects: objectsToDelete, Quiet: true },
+  });
+  while (numTries <= MAXTRIES) {
+    try {
+      const returned = await storageClient.send(deleteObjectsCommand);
+      if (!returned["Deleted"]) {
+        return { success: true };
+      }
+    } catch (error) {
+      //TODO: Write cron job to mark unsuccessful deletions to delete later
+      throw new Error("All attachments not successfully deleted");
+    }
+    numTries++;
+  }
```

Comment:
Update this loop to take the form:
```js
let count = 0;
const maxTries = 3;
while (true) {
  try {
    logic
  } catch (e) {
    if (numTries++ === maxTries) throw new Error(...)
  }
}
```
Right now we throw an error on the first try - we want to only throw an error after we have reached out maximum number of tries.